sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -903,6 +904,28 @@ def test_non_euclidean_kneighbors():
                   radius, metric='euclidean')
 
 
+def test_kneighbors_parallel():
+    X, y = datasets.make_classification(n_samples=10, n_features=2,
+                                        n_redundant=0, random_state=0)
+    X_train, X_test, y_train, y_test = train_test_split(X, y)
+    for algorithm in ALGORITHMS:
+        clf = neighbors.KNeighborsClassifier(n_neighbors=3,
+                                             algorithm=algorithm)
+        clf.fit(X_train, y_train)
+        y_1 = clf.predict(X_test)
+        dist_1, ind_1 = clf.kneighbors(X_test)
+        A_1 = clf.kneighbors_graph(X_test, mode='distance').toarray()
+        for n_jobs in [-1, 2, 5, 15]:
```

Comment:
Is there a good reason for the '15' in this list?

If not, I'd rather remove it: it will load the host machine. In addition, removing it will make the tests a little bit faster, which is always a good idea.